src/ngraph/runtime/interpreter/opset_int_tbl.hpp

```diff
@@ -28,4 +28,5 @@ NGRAPH_OP(LogicalNot, op::v1)
 
 #define ID_SUFFIX(NAME) NAME##_v3
 NGRAPH_OP(ShapeOf, op::v3)
```

Comment:
Let's take this little section our and remove the _v3 enums from the switch cases. They were only added because we needed a way to get the interpreter to run a couple new ops.

Reply:
Without adding this, it has build error. This is the same as ShapeOf v3.